packages/circuit-ui/components/TextArea/TextArea.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Can we avoid to disable this for the entire file? Happy to keep it for the relevant lines if the rule is really bothering us here

Reply:
I removed it,
Apparently, it's not needed anymore, thanks for seeing this !